YogurtTheBot.Game.Server/appsettings.json

```diff
@@ -7,12 +7,14 @@
     }
   },
   "MongoSettings": {
-    "Database": "sb",
-    "ConnectionString": "mongodb://localhost:27017",
+    "Database": "root",
```

Comment:
I believe this should be in separate config. appsettings.Dockerfile.json or something like that, so this won't broke rgular development